src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -17,372 +17,489 @@ public static class Sse
 
         /// <summary>
         /// __m128 _mm_max_ss (__m128 a,  __m128 b)
+        ///   MAXSS xmm, xmm/m32
         /// </summary>
         public static Vector128<float> MaxScalar(Vector128<float> left, Vector128<float> right) => MaxScalar(left, right);
 
         /// <summary>
         /// __m128 _mm_min_ps (__m128 a,  __m128 b)
+        ///   MINPS xmm, xmm/m128
         /// </summary>
         public static Vector128<float> Min(Vector128<float> left, Vector128<float> right) => Min(left, right);
 
         /// <summary>
         /// __m128 _mm_min_ss (__m128 a,  __m128 b)
+        ///   MINSS xmm, xmm/m32
         /// </summary>
         public static Vector128<float> MinScalar(Vector128<float> left, Vector128<float> right) => MinScalar(left, right);
 
         /// <summary>
         /// __m128 _mm_move_ss (__m128 a, __m128 b)
+        ///   MOVSS xmm, xmm
         /// </summary>
         public static Vector128<float> MoveScalar(Vector128<float> upper, Vector128<float> value) => MoveScalar(upper, value);
 
         /// <summary>
         /// __m128 _mm_movehl_ps (__m128 a,  __m128 b)
+        ///   MOVHLPS xmm, xmm
         /// </summary>
         public static Vector128<float> MoveHighToLow(Vector128<float> left, Vector128<float> right) => MoveHighToLow(left, right);
 
         /// <summary>
         /// __m128 _mm_movelh_ps (__m128 a,  __m128 b)
+        ///   MOVLHPS xmm, xmm
         /// </summary>
         public static Vector128<float> MoveLowToHigh(Vector128<float> left, Vector128<float> right) => MoveLowToHigh(left, right);
 
         /// <summary>
         /// int _mm_movemask_ps (__m128 a)
+        ///   MOVMSKPS reg, xmm
         /// </summary>
         public static int MoveMask(Vector128<float> value) => MoveMask(value);
 
         /// <summary>
         /// __m128 _mm_mul_ps (__m128 a, __m128 b)
+        ///   MULPS xmm, xmm/m128
         /// </summary>
         public static Vector128<float> Multiply(Vector128<float> left, Vector128<float> right) => Multiply(left, right);
 
         /// <summary>
         /// __m128 _mm_mul_ss (__m128 a, __m128 b)
+        ///   MULPS xmm, xmm/m32
         /// </summary>
         public static Vector128<float> MultiplyScalar(Vector128<float> left, Vector128<float> right) => MultiplyScalar(left, right);
 
         /// <summary>
         /// __m128 _mm_or_ps (__m128 a,  __m128 b)
+        ///   ORPS xmm, xmm/m128
         /// </summary>
         public static Vector128<float> Or(Vector128<float> left, Vector128<float> right) => Or(left, right);
 
+        /// <summary>
+        /// void _mm_prefetch(char* p, int i)
+        ///   PREFETCHT0 m8
+        /// </summary>
+        public static unsafe void Prefetch0(void* address) => Prefetch0(address);
+
+        /// <summary>
+        /// void _mm_prefetch(char* p, int i)
+        ///   PREFETCHT1 m8
+        /// </summary>
+        public static unsafe void Prefetch1(void* address) => Prefetch1(address);
```

Comment:
Shall we update other Load* intrinsics to `void*`?
For example,
```csharp
//Avx
        public static unsafe Vector256<sbyte> LoadVector256(sbyte* address) => LoadVector256(address);
        public static unsafe Vector256<byte> LoadVector256(byte* address) => LoadVector256(address);
        public static unsafe Vector256<short> LoadVector256(short* address) => LoadVector256(address);
        public static unsafe Vector256<ushort> LoadVector256(ushort* address) => LoadVector256(address);
        public static unsafe Vector256<int> LoadVector256(int* address) => LoadVector256(address);
        public static unsafe Vector256<uint> LoadVector256(uint* address) => LoadVector256(address);
        public static unsafe Vector256<long> LoadVector256(long* address) => LoadVector256(address);
        public static unsafe Vector256<ulong> LoadVector256(ulong* address) => LoadVector256(address);
        public static unsafe Vector256<float> LoadVector256(float* address) => LoadVector256(address);
        public static unsafe Vector256<double> LoadVector256(double* address) => LoadVector256(address);
```
=>
```csharp
public static unsafe Vector256<T> LoadVector256<T>(void* address) => LoadVector256<T>(address);
```

cc @jkotas @CarolEidt

Reply:
In this case we load into vector of predefined type and enforcing pointer to be of the same type seems correct

Reply:
Agree, thanks!